### PR TITLE
Rescue FaradayMiddleware::RedirectLimitReached

### PR DESCRIPTION
**Pull request by dentarg**
Same as #153 but for the new branch.

### CODE REVIEW

File: spec/meta_inspector/redirections_spec.rb [hunk 1]
```diff
@@ -18,6 +18,25 @@
...
+
+      before do
+        12.times { |i| register_redirect(i, i+1) }
+      end
+
+      let(:perform_request) do
+        MetaInspector.new("http://example.org/1")
+      end
+
+      it "it raises an error" do
```
Comment (jaimeiniesta) [inline spec/meta_inspector/redirections_spec.rb]: This reads a bit weird, I would leave it just like `it "raises an error"`.

File: spec/meta_inspector/redirections_spec.rb [hunk 2]
```diff
@@ -18,6 +18,25 @@
...
+        expect {
+          perform_request
```
Comment (jaimeiniesta) [inline spec/meta_inspector/redirections_spec.rb]: As we're not using this `perform_request` in more places, I would remove it and just use `MetaInspector.new("http://example.org/1")` directly, this way we can remove the 3 lines required to its definition above.

**Comment by jaimeiniesta** [comment]
:+1: just check the minor corrections noted, please.

**Comment by dentarg** [comment]
@jaimeiniesta done!

**Comment by jaimeiniesta** [comment]
Thanks!

**Pull request merged**